File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function(allowed, words) {
    let count=0
    for(i=0;i<words.length;i++){
        let isValid=true;
        for(j=0;j<words[i].length;j++){
            if(!allowed.includes(words[i][j])){
                isValid=false
                break;
            }
        }if(isValid){
            count++
        }
        
    }
    return count
    
};

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.py
class Solution(object):
    def countConsistentStrings(self, allowed, words):
        count=0
        for word in words:
            is_Valid=True
            for ch in word:
                if not ch in allowed:
                    is_Valid=False
                    break
            if is_Valid:
                count+=1
        return count
        